app/src/main/java/com/example/plantsapp/presentation/ui/notification/TaskNotificationManager.kt

```diff
@@ -78,11 +85,29 @@ class TaskNotificationManager @Inject constructor(
         )
     }
 
+    private fun createCompletePendingIntent(task: Task, notificationId: Int): PendingIntent {
+        val completeIntent = Intent(context, NotificationBroadcastReceiver::class.java)
+            .apply {
+                this.notificationId = notificationId
+                taskId = task.id
+                taskFrequency = task.frequency
+                taskKey = TaskKeys.from(task).key
+            }
+        return PendingIntent.getBroadcast(
+            context,
+            completeRequestCode++,
```

Comment:
also, probably you can move completeRequestCode into constant, and do not change it every time 

If I get it right, it is not used anywhere else right now?